get-started/quickstart.rst

```diff
@@ -0,0 +1,383 @@
+specify ``local.zeek`` in the command line in order to load its contents.
+
+Run ``zeekctl`` in order to get into a new prompt:
+
+.. code-block:: console
+
+     $ zeekctl
+     Hint: Run the zeekctl "deploy" command to get started.
+
+     Welcome to ZeekControl 2.5.0-76
+
+     Type "help" for help.
+
+     [ZeekControl] >
+
+Then run ``deploy`` to get started:
+
+.. code-block:: console
+
+     [ZeekControl] > deploy
+
+In another terminal, run the same two curl commands from before:
+
+.. code-block:: console
+
+     $ curl -X GET http://zeek.org
+     $ curl -X WEIRD http://zeek.org
+
+Then return to the ZeekControl prompt and stop it:
+
+.. code-block:: console
+
+     [ZeekControl] > stop
+     stopping zeek ...
+     [ZeekControl] > exit
+
+And exit from ``zeekctl``:
+
+.. code-block:: console
+
+     [ZeekControl] > exit
+
+The logs from ZeekControl will not appear in your current directory. Instead,
+they will appear in ``$PREFIX/logs/current`` when running. Since the process was
+stopped, they will appear in a directory with the current date within 
+``$PREFIX/logs/`` - such as ``$PREFIX/logs/2025-01-01/``.
+
+These logs are compressed as ``.log.gz`` files from gzip_. You may decompress
+these via ``gunzip`` then read them, or use gzip’s packaged ``zcat`` command.
+On Mac, this looks like:
+
+.. code-block:: console
+
+     $ zcat < $PREFIX/logs/2025-01-08/weird.11:03:38-11:03:43.log.gz
+     <...>
+     1736352218.157077       CFvENWVlkwVHhLL35       2603:6081:18f0:99e0:7da2:6b81:9a83:cb4e 57823   2606:2800:21f:cb07:6820:80da:af6b:8b2c   80      unknown_HTTP_method     WEIRD   F       zeek    -
+
+The logs contain the ``WEIRD`` HTTP request.
+
+More information on using ZeekControl can be found in the
+`ZeekControl documentation`_. More information on setting up a cluster can be
+found in the :doc:`cluster-setup` section.
+
+Clusters
+========
+
+ZeekControl is also used to manage a cluster of Zeek processes. This cluster
+contains many processes which analyze traffic together. Clusters are normally
+split among many machines. For this example, all nodes will be local.
+
+First, return to the ``$PREFIX/etc/node.cfg`` configuration file. It currently
+contains one "standalone" node: 
+
+.. code-block:: console
+
+     [zeek]
+     type=standalone
+     host=localhost
+     interface=en0
+
+
+A standalone node is not in a cluster. Instead, this will change to multiple
+nodes which work together. The following configuration is commented out in the
+``node.cfg`` file by default. Remove the ``[zeek]`` node from above and paste
+this into the file (or uncomment the code):
+
+.. code-block:: console
+
+     [logger]
+     type=logger
+     host=localhost
+
+     [manager]
+     type=manager
+     host=localhost
+
+     [proxy]
+     type=proxy
+     host=localhost
+
+     [worker-1]
+     type=worker
+     host=localhost
+     interface=en0
+
+     [worker-2]
+     type=worker
+     host=localhost
+     interface=en0
```

Comment:
Just using a single worker might be better in quickstart. There's too many caveats with multiple workers (proper, load-balancing, af_packet_fanout_id, ...). I'm even a bit worried that running through zeekctl is overwhelming, but there's also no real way around it for a real deployment.

Reply:
I agree, I only kept that one because it's commented out by default, so I don't need to tell people to only comment one out. That could cause frustration if they are only *kind of* following the guide.

Maybe I just remove mentions of uncommenting altogether - but then people might see them anyway. I'll grapple with this in my head for a bit to see if there's something cleaner, or maybe those wouldn't be issues?